src/main/resources/application.yml

```
@@ -23,4 +23,8 @@ springdoc:
     tags-sorter: alpha
     operations-sorter: alpha
   cache:
-    disabled: true
\ No newline at end of file
+    disabled: true
+
+jwt:
+  secret-key: dGhpcy1pcy1hLXNlY3JldC1rZXktZm9yLWp3dC10b2tlbi1hbmQtd2ViZmx1eA==
+  expiration-time: 3600000
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)